R/minimize.R

```diff
@@ -115,6 +115,48 @@ print.adoptrOptimizationResult <- function(x, ...) {
 
 
 
+#' Initial design
+#'
+#' The optimization method \code{\link{minimize}} requires an initial
+#' design for optimization.
+#' The function \code{get_initial_design} provides an initial guess based on a
+#' fixed design for a z-test that fulfills constraints on type I error rate and power.
+#' Note that a situation-specific initial design may be much more efficient.
+#'
+#' @param theta the alternative effect size
+#' @param alpha maximal type I error rate
+#' @param beta maximale type II error rate
+#' @param type is a two-stage, group-sequential, or one-stage design requried?
+#' @param order desired integration order
+#'
+#' @examples
+#' init <- get_initial_design(
+#'    theta = 0.3,
+#'    alpha = 0.025,
+#'    beta  = 0.2,
+#'    type  = "two-stage",
+#'    order = 7L
+#' )
+#'
+#' @export
+get_initial_design <- function(theta, alpha = 0.025, beta = 0.2,
+                               type = c("two-stage", "group-sequential", "one-stage"),
+                               order = 7L) {
+    type <- match.arg(type)
+    if(alpha <= 0 || alpha >= 1 || beta <= 0 || beta >= 1)
```

Comment:
please try to follow common R style patterns, i.e. space after if (I know it's a function, but that makes code a lot easier to read) cf. https://style.tidyverse.org/syntax.html#parentheses